pyodide-build/pyodide_build/install_xbuildenv.py

```diff
@@ -112,8 +118,16 @@ def install(path: Path, *, download: bool = False, url: str | None = None) -> No
     from . import __version__
 
     version = __version__
+
     if download:
```

Comment:
A better default behavior might be to check if it's downloaded already maybe? To avoid re-downloading this at each `pyodide build` command.